### DISCUSSION TITLE
Naive-UI vscode theme

### DISCUSSION THREAD

**Discussion by brys0** [tusen-ai/naive-ui#3518] [Show and tell]
Hey everyone! I made a VSCode theme inspired by Naive-UI's wonderful color palette 

[Repo](https://github.com/brys0/vscode-naive-ui-theme)
[VSIX Download](https://github.com/brys0/vscode-naive-ui-theme/releases/download/v0.0.1/vscode-navive-ui-theme-0.0.1.vsix)
![dev preview](https://user-images.githubusercontent.com/35676267/184657712-10a2b385-fbe7-4ea9-a2ea-5bf11048fdbd.png)
![real preview](https://user-images.githubusercontent.com/35676267/184657843-b583a76f-5683-49b0-bc53-3ce11c0b52a2.png)

In the future I will probably add Naive-UI's light theme as well. But, for now you have just the dark mode.

**Comment by Sepush** [comment]
looks cool

**Comment by MD-Student** [comment]
It looks very cool. Look forward to the light version.

**Comment by jahnli** [comment]
:+1:
